tests/playwright-tests/utils/env.ts

```diff
@@ -0,0 +1,7 @@
+import dotenv from 'dotenv';
```

Comment:
This is great - I would just add a mention of having to add a .env file to the README if it's mandatory to do so.

We generally check in a `.env.example` file with local settings in so that people can just copy that to a new `.env` file - we then add something like the following to `.gitignore`:

```
.env
.env.*
!.env.example
```

For Robot, for example, we have these lines in `.gitignore` in the root of the project:

```
/tests/robot-tests/.env.*
!/tests/robot-tests/.env.example
```

Reply:
Hi @duncan-at-hiveit - Thanks for spotting this one:).  Could you please help me if you would like me to put any variables inside the '.env.example' file